scrapy_jsonrpc/jsonrpc.py

```diff
@@ -58,10 +58,11 @@ def jsonrpc_server_call(target, jsonrpc_request, json_decoder=None):
         json_decoder = ScrapyJSONDecoder()
 
     try:
-        req = json_decoder.decode(jsonrpc_request)
+        req = json_decoder.decode(jsonrpc_request.decode())
```

Comment:
`.decode()` is almost always wrong - it decodes using `sys.getdefaultencoding()`, which can vary across machines, and which is often ascii. I think it is better to decode from UTF-8 because this is what JSON standard requires.